rules/azure/defense_evasion_azure_conditional_access_policy_modified.toml

```diff
@@ -0,0 +1,44 @@
+[metadata]
+creation_date = "2020/09/01"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/09/01"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies when an Azure Conditional Access policy is modified. Azure Conditional Access policies control access to
+resources via if-then statements. For example, if a user wants to access a resource, then they must complete an action
+such as using multi-factor authentication to access it. An adversary may modify a Conditional Access policy in order to
+weaken their target's security controls.
+"""
+from = "now-25m"
+index = ["filebeat-*"]
+language = "kuery"
+license = "Elastic License"
+name = "Azure Conditional Access Policy Modified"
+note = "The Azure Filebeat module must be enabled to use this rule."
+references = ["https://docs.microsoft.com/en-us/azure/active-directory/conditional-access/overview"]
+risk_score = 47
+rule_id = "bc48bba7-4a23-4232-b551-eca3ca1e3f20"
+severity = "medium"
+tags = ["Elastic", "Azure", "Continuous Monitoring", "SecOps", "Configuration Audit"]
+type = "query"
+
+query = '''
+event.module:azure and event.dataset:azure.activitylogs and event.category:Administrative and azure.activitylogs.operation_name:"Update policy" and event.outcome:success
```

Comment:
I'm updating this query, as this event appears in both the Azure audit logs and activity logs (good find @bm11100).

```suggestion
event.module:azure and (event.dataset:azure.activitylogs or event.dataset:azure.auditlogs) and (azure.activitylogs.operation_name:"Update policy" or azure.auditlogs.operation_name:"Update policy") and event.outcome:success
```

<img width="2553" alt="image" src="https://user-images.githubusercontent.com/56409778/93897534-6a3bc600-fcaf-11ea-8512-30503e0cd443.png">

Reply:
I dont think the Audit Logs use `event.outcome` so we could do the below
```
event.module:azure and event.dataset:(azure.activitylogs or azure.auditlogs) and (azure.activitylogs.operation_name:"Update policy" or azure.auditlogs.operation_name:"Update policy")
```

Reply:
Actually i see the comment above, this was fixed.